src/brogue/Combat.c

```diff
@@ -931,7 +931,7 @@ void applyArmorRunicEffect(char returnString[DCOLS], creature *attacker, short *
             if (rand_percent(10)) {
                 rogue.armor->strengthRequired++;
                 sprintf(returnString, "your %s suddenly feels heavier!", armorName);
-                equipItem(rogue.armor, true, NULL);
+                equipItem(rogue.armor, true, NULL, false);
```

Comment:
I guess this is to update all bonuses

Reply:
I think the various call sites with `false` are there to recalculate bonues and other passive quantities, yea.